api_spec.yaml

```diff
@@ -1832,6 +1752,89 @@ paths:
                                     - Unauthorised access
+    /dashboard/api/tenants/core-config/list:
+        get:
+            tags:
+                - Multitenancy
+            summary: Gets all the possible core config options.
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    oneOf:
+                                        - allOf:
+                                              - $ref: "#/components/schemas/coreConfigType"
+                                              - type: object
+                                                properties:
+                                                    type:
+                                                        type: string
+                                                        enum:
+                                                            - string
+                                                            - boolean
+                                                            - number
+                                                        description: Data type of the property
+                                        - allOf:
+                                              - $ref: "#/components/schemas/coreConfigType"
+                                              - type: object
+                                                properties:
+                                                    type:
+                                                        type: string
+                                                        enum:
+                                                            - enum
+                                                        description: Data type of the property
+                                                    options:
+                                                        type: array
+                                                        items:
+                                                            type: string
+                                                        description: List of possible values for the property in case it is of enum type
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
+    /dashboard/api/tenants/built-in-factors/list:
```

Comment:
/dashboard/api/mfa/built-in-factors/list

Reply:
Done